### PR TITLE
Updates to Tui tree query, and fix typo, unused imports, etc

### PR DESCRIPTION
**Pull request by kinow**
This is a small change with no associated Issue.

@oliver-sanders just checking if it's worth to add a few things we have from the Cylc UI tree query on Tui. WDYT?

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Already covered by existing tests.
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.

### CODE REVIEW

File: cylc/flow/tui/data.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
           id
           name
         }
-        jobs {
+        jobs(sort: { keys: ["submit_num"], reverse: true}) {
```
Comment (kinow) [inline cylc/flow/tui/data.py:23]: i.e. same as doing `.sort(lambda..., reverse=True)` but on the server side

File: cylc/flow/tui/data.py [hunk 2]
```diff
@@ -33,7 +33,7 @@
           meanElapsedTime
         }
       }
-      familyProxies(states: $taskStates) {
+      familyProxies(exids: ["root"], states: $taskStates) {
```
Comment (kinow) [inline cylc/flow/tui/data.py:36]: Eliminates `if familyProxy == root`. Only place where root is used—I think— is when computing cycle points. It's also used when it's the first parent, but that data is still in the response.

File: cylc/flow/tui/util.py [hunk 3]
```diff
@@ -115,27 +106,24 @@ def compute_tree(flow):
...
+                first_parent
+                and first_parent['name'] != 'root'
+        ):
+            parent_node = add_node(
+                'family', first_parent['id'], nodes)
+            parent_node['children'].append(family_node)
+        else:
+            add_node(
+                'cycle', idpop(family['id']), nodes
+            )['children'].append(family_node)
```
Comment (kinow) [inline cylc/flow/tui/util.py:126]: Didn't change anything above, except for removing the `if family['name'] != 'root':` and indentation.

File: cylc/flow/tests/tui/test_util.py [hunk 4]
```diff
@@ -160,12 +153,6 @@ def test_compute_tree():
             }
         ],
         'familyProxies': [
-            {  # root family node
-                'name': 'root',
-                'id': '1|root',
-                'cyclePoint': '1',
-                'firstParent': None
-            },
```
Comment (oliver-sanders) [inline cylc/flow/tests/tui/test_util.py:168]: This was in here to ensure that the root node got filtered if present.

BTW this whole method will need to be re-implemented for deltas.

File: cylc/flow/tui/data.py (hunk 1 above)
Comment (oliver-sanders) [reply cylc/flow/tui/data.py:23]: Not sure whether it's better to put load on the scheduler or on Tui.

Perhaps the data store could keep jobs in reverse order for efficiency.

File: cylc/flow/tests/tui/test_util.py [hunk 5]
```diff
@@ -217,14 +204,20 @@ def test_compute_tree():
...
             }
         ]
     })
+    """It computes a tree in the right structure for urwid.
+
+    Note this test doesn't use full data or propper ids because it's
+    purpose is not to test the GraphQL interface but to ensure the
+    assumptions made by compute_tree check out.
+
+    """
```
Comment (oliver-sanders) [inline cylc/flow/tests/tui/test_util.py:220]: Why has this docstring moved?

Comment (kinow) [reply cylc/flow/tests/tui/test_util.py:220]: Argh, accidental move, sorry. Will fix it tomorrow morning :+1:

File: cylc/flow/tests/tui/test_util.py (hunk 4 above)
Comment (kinow) [reply cylc/flow/tests/tui/test_util.py:168]: Yup, removed as the data is filtered in the server side in this PR like in Cylc UI. Probably safe, as the root family shouldn't be present any longer I think.

**Comment by oliver-sanders** [comment]
I can't poke Travis to push this through...

**Comment by dwsutherland** [comment]
Kicked Travis 🥾

**Pull request merged**